app/src/main/java/com/example/bookwormadventuresdeluxe2/GlobalApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.app.Application;
+import android.content.Context;
+
+//https://stackoverflow.com/questions/2002288/static-way-to-get-context-in-android
+public class GlobalApplication extends Application
+{
+    private static Context context;
+
+    public void onCreate()
+    {
+        super.onCreate();
+        GlobalApplication.context = getApplicationContext();

```

Comment:
You can't call getApplicationContext() from Book since Book is not an Activity. Getting the context in Book proved to be a massive challenge and this was the only solution that worked. And with this solution, we are able to get the resources from anywhere in the application.